src/main/resources/templates/aeolus/java/plain_gradle.yaml

```
@@ -0,0 +1,26 @@
+api: v0.0.1
+metadata:
+    name: java-gradle
+    id: java-windfile
+    description: Build Plan for a java Exercise
+    author: Andreas Resch
+    gitCredentials: artemis_admin
+    docker:
+        image: ls1tum/artemis-maven-template
+        tag: java17-20
+        volumes:
+            - ${WORKDIR}:${WORKDIR}
+            - ${TMPDIR}:${TMPDIR}
+        parameters:
+            - --cpus
+            - '"2"'
+            - --memory
+            - '"2g"'
+            - --memory-swap
+            - '"2g"'
+            - --pids-limit
+            - '"1000"'
+actions:
+    - name: gradle
+      script: ./gradlew clean test
```

Do you think it would be possible to add a line to set file permissions? In Local CI it only works like this and I don't think it would have any implications on bamboo or jenkins
```suggestion
      script: |
         chmod +x gradlew
         ./gradlew clean test
```